examples/README.md

```diff
@@ -0,0 +1,11 @@
+# Examples
+
+## Project setup
+```
+composer install
```

Comment:
Should this be executed from examples directory or project root?

Reply:
We need to be at the root of the project. Added clarification README.md